apps/chat/src/pages/index.tsx

```diff
@@ -111,6 +113,13 @@ export default function Home({ initialState }: HomeProps) {
         }),
       );
     }
+    if (searchParams.has(ISOLATED_MODEL_QUERY_PARAM)) {
+      dispatch(
```

Comment:
set this setting state inside getServerSideProps

Reply:
@mikitabut , `window is not defined` into `getServerSideProps`